src/pharmpy/modeling/parameter_sampling.py

```diff
@@ -85,6 +85,9 @@ def _sample_from_function(
     """
     rng = create_rng(rng)
 
+    parameter_estimates = parameter_estimates[
```

Comment:
There is a better way of doing this. Use `model.parameters.fixed` eller `model.parameters.nonfixed

Reply:
updated to use model.parameters.nonfixed instead